config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  
```

Comment:
like how you reorganized the routes page

Reply:
I agree! Looks great.